components/overview/RedeemUnbonded.js

```diff
@@ -198,6 +202,16 @@ const RedeemUnbonded = withSlideIn(
 								<>
 									<h3 className="mt-4 text-2xl">Confirmation</h3>
 									<div className="w-full mt-8">
+										{controllerAccount &&
+											controllerBalances &&
+											transactionFee +
+												apiInstance?.consts.balances.existentialDeposit.toNumber() >
```

Comment:
change this to transactionFee + (apiInstance?.consts.balances.existentialDeposit.toNumber() / 2)